pavelib/tests.py

```diff
@@ -134,6 +138,9 @@ def test_lib(options):
     make_option("-q", "--quiet", action="store_const", const=0, dest="verbosity"),
     make_option("-v", "--verbosity", action="count", dest="verbosity", default=1),
     make_option("--pdb", action="store_true", help="Drop into debugger on failures or errors"),
+    make_option(
+        "--enable-migrations", action="store_true", dest="enable_migrations", help="Run migrations instead of syncdb"
```

Comment:
help text should also point out that this can be set with an environment variable.